24_clustering.Rmd

```diff
@@ -402,30 +370,23 @@ euclidean_dmm_plot <- ggplot(
 euclidean_dmm_plot
 ```
 
-## Community Detection
+## Graph-based clustering
 
 Another approach for discovering communities within the samples of the
 data, is to run community detection algorithms after building a
 graph. The following demonstration builds a graph based on the k
 nearest-neighbors and performs the community detection on the fly.
 
-_`bluster`_ [@R_bluster] package offers several clustering methods,
-among which graph-based are present, enabling the community detection
-task.
-
-Installing package:
-
-```{r}
-library(bluster)
-```
+Here, we will be using the `cluster` function with a graph-based clustering 
+function, enabling the community detection task.
```

Comment:
Is there some recent microbiome literature showing that this method has some advantages compared to the other clustering methods? Otherwise we could potentially remove this example case to keep contents focused.